csharp_dotnetcore/3.Welcome-Users/WelcomeUser.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <CodeAnalysisRuleSet>..\samples.ruleset</CodeAnalysisRuleSet>
```

Comment:
We've been moving / copying the ruleset into each sample and giving it a sample related name.